tests/test_processing_common.py

```diff
@@ -84,9 +91,7 @@ def prepare_image_inputs(self):
         """This function prepares a list of PIL images, or a list of numpy arrays if one specifies numpify=True,
         or a list of PyTorch tensors if one specifies torchify=True.
```

Comment:
should the docstring be more specific as the method doesn't accept `numpify` and `torchify` as args? Here and in overwritten methods in ModelTesters

Reply:
Updated!